packs/octopusdeploy/actions/get_deployments.py

```diff
@@ -0,0 +1,12 @@
+from lib.actions import OctopusDeployAction
+
+__all__ = [
+    'GetDeploymentsAction'
+]
+
+
+class GetDeploymentsAction(OctopusDeployAction):
+    def run(self, project_id):
+        result = self.make_get_request(action="deployments?projects="
+                                       % project_id)
+        return result['Items']
```

Comment:
Just noting that irrespective of what status code the HTTP request produces this action will always pass. If you want the action to fail on non 2XX code, you'll need the base method to return status and throw an exception to fail the action. 

Also, I highly recommend using .get('Items', None) :P.